harvester/logger_config.py

```diff
@@ -0,0 +1,25 @@
+LOGGING_CONFIG = {
+    "version": 1,
+    "formatters": {
+        "standard": {
+            "format": (
+                "[%(asctime)s] %(levelname)s "
+                "[%(name)s.%(funcName)s:%(lineno)d] %(message)s"
+            )
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": "INFO",
```

Comment:
Someday we may change this up to "WARN", however for today more logging is better.